backend/src/auth/auth.controller.ts

```diff
@@ -1,6 +1,7 @@
 import * as bcrypt from 'bcrypt';
 import { NextFunction, Request, Response } from 'express';
 import * as status from 'http-status';
+import { randomUUID } from 'crypto';
```

Comment:
```suggestion
import { randomUUID } from 'node:crypto';
```

 [randomUUID가](https://nodejs.org/docs/latest-v18.x/api/crypto.html#cryptorandomuuidoptions) node 빌트인 모듈 임을 [명시적으로 표시](https://nodejs.org/docs/latest-v18.x/api/esm.html#node-imports)해주면 좋을 것 같습니다